### ISSUE TITLE
Making forms accessible and conform to PMRO class naming

### ISSUE THREAD

**Issue by patric-boehner** [strangerstudios/pmpro-reason-for-cancelling#27]
**Describe the bug**
The current reason for the cancellation form not being accessible is that it lacks a label and association with the form field. For screen reader users, the purpose of the form has no description.

The form and form fields also lack class names used elsewhere in PMPRO forms, which breaks styling that might already be using PMPR's naming convention.

This is probably not the right place for this but here is an example commit to the change: https://github.com/strangerstudios/pmpro-reason-for-cancelling/commit/156e05912e61515b6ef9acd2d93e25ad07d4fe47

**Expected behavior**
An HTML label element with a 'for' attribute that matches the id for the textarea element.

**Isolating the problem (mark completed items with an [x]):**
- [x] I have deactivated other plugins and confirmed this bug occurs when only Paid Memberships Pro plugin is active.
- [x] This bug happens with a default WordPress theme active, or [Memberlite](https://wordpress.org/themes/memberlite/).
- [x] I can reproduce this bug consistently using the steps above.

**WordPress Environment**
<details>
```
Local by flywheel Version 8.1.0+6514
WP: 6.42
PHP version 8.2.10
Nginx
PMPRO Version 2.12.6
Reason For Cancelling Add On Version 0.2.1
```
</details>

**Comment by kimcoleman** [comment]
This is resolved in v1.1
